### PR TITLE
feat: improve build

### PR DESCRIPTION
**Pull request by revant**
use nodejs v14
allow build args to pass repo and branch
add WORKER_CLASS under erpnext-python in docker-compose.yml

### CODE REVIEW

**Review by Thunderbottom — APPROVED** [review]
LGTM!

File: build/bench/Dockerfile [hunk 1]
```diff
@@ -87,13 +91,12 @@ RUN bash -c "bench --version"
 # !!! UPDATE NODEJS PERIODICALLY WITH LATEST VERSIONS !!!
 # https://nodejs.org/en/about/releases/
 # https://nodejs.org/download/release/latest-v10.x/
-# https://nodejs.org/download/release/latest-v12.x/
-# https://nodejs.org/download/release/latest-v13.x/
-ENV NODE_VERSION=12.20.0
-ENV NODE_VERSION_FRAPPEV11=10.23.0
+# https://nodejs.org/download/release/latest-v14.x/
+ENV NODE_VERSION=14.17.0
+ENV NODE_VERSION_FRAPPEV11=10.24.1
```
Comment (PeterDaveHello) [inline build/bench/Dockerfile:96]: node.js v10 is EOL already, ref: https://github.com/nodejs/Release/#end-of-life-releases, maybe should also update it to v12 (consider to keep two versions of node.js here, ref: https://github.com/frappe/frappe_docker/issues/123#issuecomment-596211931)? I can send a pull request if it make sense to you.

Comment (revant) [reply build/bench/Dockerfile:96]: I think it's needed for old v11 development.
ERPNext V11 won't work without nodejs v10.

Comment (PeterDaveHello) [reply build/bench/Dockerfile:96]: Oh, okay, I got it. Thanks.

**Pull request merged**